sui/typescript/backend_examples/src/invisible_wallet.ts

```diff
@@ -104,55 +98,16 @@ async function sponsorSignExecuteInThreeRequests(signer: ShinamiWalletSigner,
   //  `sponsorTransactionBlock` along with the sender's signature 
   const executeSponsoredTxResponse = await nodeClient.executeTransactionBlock({
     transactionBlock: sponsoredResponse.txBytes,
-    signature: [senderSignature.signature, sponsoredResponse.signature],
-    options: { showEffects: true },
```

Comment:
Same as above

Reply:
Intentionally removed for the same reason